03_user_and_group_management/050_configure_pam/051_pam.md

```diff
@@ -0,0 +1,19 @@
+# Pluggable Autentication Modules
+This chapter is based on the [Red Hat Documentation](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/system-level_authentication_guide/pluggable_authentication_modules).
+Pluggable Authentication Modules (PAM) central way to authenticate. PAMs offer different types of authentication (Kerberos, NIS, local filesystem) and are therefor called pluggable. 
+
+Because of its pluggable nature PAM gives administrators great flexibility on configuring authentication on systems.
+PAM can be used by a wide range of applications.
+
+On RedHat Systems the tools `authconfig` or `authselect` are recommended to configure PAM, on Ubuntu the configuration files are edited directly.
+
+The configurations files are `/etc/pam.conf` or any file in `/etc/pam.d` (if `/etc/pam.d` exists `/etc/pam.conf` will be ignored).
+Each of these files should consist of multiple rule lines. Each rule-line has a syntax as follows:
+
+```
+service type control module-path module-arguments
```

Comment:
Could you provide one simple example?

Reply:
Provided an excerpt of the /etc/pam.d/login file as an example.